libraries/MySensors/MySensor.cpp

```diff
@@ -102,7 +102,7 @@ void MySensor::begin(void (*_msgCallback)(const MyMessage &), uint8_t _nodeId, b
 
 	// Setup radio
 	if (!radio.init()) {
-		debug(PSTR("radio initialization failed!"));
+		debug(PSTR("radio initialization failed!\n"));
```

Comment:
I could do it if you want. ;)

But the fundamental question is what we want here? Is it of any value to continue? In some use cases it might be but I cannot come up with one now;). Hmm... In this case the radio is probably mis-wired and we should probably just stop the execution to alert the user as it is none-recoverable.